src/schema/nmdc.yaml

```diff
@@ -82,6 +82,7 @@ prefixes:
   igsn: https://app.geosamples.org/sample/igsn/
   img.taxon: "https://bioregistry.io/img.taxon:"
   jgi.proposal: "https://bioregistry.io/jgi.proposal:"
+  jgi.analysis: http://example.org/jgi/analysis/
```

Comment:
prefix expansion value should be "https://data.jgi.doe.gov/search?q=" instead of http://example.org/jgi/analysis/ example.org is a dummy expansion url we use when prefixes are not resolvable.